packages/blockchain-wallet-v4/src/network/api/sfox/index.js

```diff
@@ -8,8 +8,20 @@ export default ({ url }) => {
   }).then(res => {
     return res
   })
+  const fetchTradesWithLength = (token, num, apiKey, prod) => {
```

Comment:
@plondon lets discuss this next week. this commit adds a V4 core API call to get trades with a query param which will get all the trades, not just 10. it also saves the trades in metadata. As you can see in fetchTrades saga I had to reimplement the creation of the trade classes from the bitcoin-*-clients. some of this code can be optimized and made more concise; this is more of a PoC.

Reply:
Yes, we should do this in bitcoin-sfox-client and possibly outside of this PR

Reply:
agree and agree

Reply:
I'm merging for now since there are a ton of other fixes in this PR but please move this like discussed.